master/internal/task/allocation_test.go

```diff
@@ -83,45 +82,41 @@ func TestAllocation(t *testing.T) {
 				Msg: sproto.ResourcesAllocated{
 					ID:           a.req.AllocationID,
 					ResourcePool: "default",
-					Reservations: reservations,
+					Resources:    resources,
 				},
 			}).Error())
 			system.Ask(rm, actors.ForwardThroughMock{To: self, Msg: actor.Ping{}}).Get()
 			require.Nil(t, trialImpl.AssertExpectations())
 			require.True(t, db.AssertExpectations(t))
 
 			// Pre-ready stage.
-			for _, r := range reservations {
-				containerStateChanged := sproto.TaskContainerStateChanged{
-					Container: cproto.Container{
-						Parent:  actor.Address{},
-						ID:      r.Summary().ID,
-						State:   cproto.Assigned,
-						Devices: []device.Device{},
-					},
+			for _, r := range resources {
+				summary := r.Summary()
+				containerStateChanged := sproto.ResourcesStateChanged{
+					ResourcesID:    summary.ResourcesID,
+					ResourcesState: sproto.Assigned,
 				}
 				db.On("UpdateAllocationState", mock.Anything).Return(nil)
 				require.NoError(t, system.Ask(self, containerStateChanged).Error())
-				containerStateChanged.Container.State = cproto.Pulling
+				containerStateChanged.ResourcesState = sproto.Pulling
 				require.NoError(t, system.Ask(self, containerStateChanged).Error())
-				containerStateChanged.Container.State = cproto.Starting
+				containerStateChanged.ResourcesState = sproto.Starting
 				require.NoError(t, system.Ask(self, containerStateChanged).Error())
-				containerStateChanged.Container.State = cproto.Running
-				containerStateChanged.ContainerStarted = &sproto.TaskContainerStarted{
+				containerStateChanged.ResourcesState = sproto.Running
+				containerStateChanged.ResourcesStarted = &sproto.ResourcesStarted{
 					Addresses: []cproto.Address{
 						{
-							ContainerIP:   "0.0.0.0",
+							ContainerIP:   "172.0.0.3",
```

Comment:
why `.3`?

Reply:
:shrug: just the first number i typed, is a test, idk why i changed this